lib/zold/upgrades.rb

```diff
@@ -0,0 +1,27 @@
+require 'semantic'
+
+module Zold
+  # Class to manage data upgrades (when zold itself upgrades).
+  class Upgrades
+    def initialize(version_file)
+      @version_file = version_file
```

Comment:
@botanicus keep in mind that compound variable names are a code smell: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html